api/utils/keypaths/keypaths.go

```diff
@@ -144,6 +146,14 @@ func CAsDir(baseDir, proxy string) string {
 	return filepath.Join(ProxyKeyDir(baseDir, proxy), casDir)
 }
 
+// TLSCAsPath returns the path to the users's TLS CA's certificates
+// for the given proxy.
+// <baseDir>/keys/<proxy>/certs.pem
+// DELETE IN 10.0. Deprecated
```

Comment:
If this needs to be removed in 10.0 shouldn't we just fix it in 9.0? Technically this code will be deprecated the moment when is merged.

Reply:
The issue is not stricte related to any `teleport` `tsh` or `tctl` flow but occurs in external tools that depends on teleport/api/client` package like  [terraform](https://github.com/gravitational/teleport-plugins/blob/master/go.mod#L20) teleport-plugin where client still uses old `tsh` that generates certs.pem dir  but the teleport-plugin uses a news version of `teleport/api/client`  that doesn't read certs from certs.pem. 

If we will be sure that all clients switched to teleport v9 where trusted certs are stored in CAS directory we will be able to drop support for `certs.pem` file.